src/theory/quantifiers/fmf/bounded_integers.cpp

```diff
@@ -786,7 +786,7 @@ bool BoundedIntegers::getBoundElements( RepSetIterator * rsi, bool initial, Node
         {
           long rr = range.getConst<Rational>().getNumerator().getLong()+1;
           Trace("bound-int-rsi")  << "Actual bound range is " << rr << std::endl;
-          for( unsigned k=0; k<rr; k++ ){
+          for( long k=0; k<rr; k++ ){
```

Comment:
Format. Can you use fixed-width types here?

Reply:
I'd rather not use fixed-width types in this PR. I wanted to keep the change minimal so that ARM64 compiles with -Werror.